src/Components/Navbar/Navbar.js

```diff
@@ -1,41 +1,62 @@
 import React from 'react'
 import './Navbar.css'
 import { Link } from "react-router-dom"
+import { useUserLogin } from "../../Context/user-login-context"
+import { useToast } from "../../Context/toast-context"
 
 function Navbar() {
-  return (
-    <div className="top-bar">
-        <div className="left-topbar-container">
-            {/* <button id="top-bar-ham-menu-btn" className="icon-btn"><i className="fa fa-bars" aria-hidden="true"></i></button> */}
-            <Link to="/">
-                <h2 className="top-bar-brand-name">Bookztron</h2>
-            </Link>
-            <div className="search-bar">
-                <input className="search-bar-input" placeholder="Search"/>
-                <button id="search-bar-btn">
-                    <i className="fa fa-search" aria-hidden="true"></i>
+
+    const { userLoggedIn, setUserLoggedIn } = useUserLogin()
+    const { showToast } = useToast()
+
+    function logoutUser()
+    {
+        localStorage.removeItem('token')
+        showToast("success","","Logged out successfully")
+        setUserLoggedIn(false)
+    }
+
+    return (
+        <div className="top-bar">
+            <div className="left-topbar-container">
+                {/* <button id="top-bar-ham-menu-btn" className="icon-btn"><i className="fa fa-bars" aria-hidden="true"></i></button> */}
```

Comment:
Remove commented code.

Reply:
It is there because I will be using this in the future.
Will remove it if I felt I won't be using it 😅